server/src/main/java/org/apache/iotdb/db/trigger/service/TriggerManagementService.java

```diff
@@ -72,6 +75,11 @@ public void register(TriggerInformation triggerInformation) throws IOException {
       acquireLock();
       checkIfRegistered(triggerInformation);
       doRegister(triggerInformation);
+    } catch (Exception e) {
+      LOGGER.warn(
+          "Failed to register trigger({}) on data node, the cause is ",
+          triggerInformation.getTriggerName(),
+          e);
```

Comment:
No need to log here, exception will be catched in createTriggerInstance and printed.